src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5373,13 +5374,28 @@ int wm_vuldet_generate_os_and_kernel_package(sqlite3 *db, agent_software *agent)
                 product_name[2] = "macos";
             }
         break;
+        case FEED_AL1:
+            product_name[0] = "linux_1";
+            product_name[1] = V_KERNEL;
```

Comment:
Better to use here `"linux"` directly to avoid confusion between a product name and the vendor for the Kernel package, even they are the same.